api/submariner/v1alpha1/submariner_types.go

```diff
@@ -58,6 +58,7 @@ type SubmarinerSpec struct {
 	NatEnabled               bool                 `json:"natEnabled"`
 	LoadBalancerEnabled      bool                 `json:"loadBalancerEnabled,omitempty"`
 	ServiceDiscoveryEnabled  bool                 `json:"serviceDiscoveryEnabled,omitempty"`
+	BrokerK8sInsecure        bool                 `json:"brokerK8sInsecure"`
```

Comment:
```suggestion
	BrokerK8sInsecure        bool                 `json:"brokerK8sInsecure, omitempty"`
```

Reply:
Yes, I wanted to see how the default was handled without that...